docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
+has not been a data loss.
 
 Parameters
 ----------
 
-`repurpose`
-  Delete excess data when a node’s roles are changed.
+* ``repurpose``
+    Delete excess data when a node’s role is changed.
+
+* ``unsafe-bootstrap``
+    Unsafely bootstrap this node as a new one-node cluster.
+
+* ``detach-cluster``
+    Specify to unsafely detach this node from its cluster so it can join
+    another cluster.
+
+* ``--ordinal <Integer>``
+    Specify which node to target if there is more than one node sharing a data
+    path. Defaults to 0, meaning to use the first node in the data path.
 
-`unsafe-bootstrap`
-  Specifies to unsafely bootstrap this node as a new one-node cluster.
+* ``-C <KeyValuePair>``
+    Configure a setting.
 
-`detach-cluster`
-  Specifies to unsafely detach this node from its cluster so it can join a
-  different cluster.
+* ``-h, --help``
+    Return all of the command parameters.
 
-`--ordinal <Integer>``
-  If there is more than one node sharing a data path then this specifies which
-  node to target. Defaults to 0, meaning to use the first node in the data path.
+* ``-s, --silent``
+    Show minimal output.
 
-`-C <KeyValuePair>`
-  Configures a setting.
+* ``-v, --verbose``
+    Show verbose output.
 
-`-h, --help`
-  Returns all of the command parameters.
+Example
+-------
+Repurposing a node as a dedicated master node
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`-s, --silent`
-  Shows minimal output.
+The following example shows you how to repurpose a former data node as a
+dedicated master node.
 
-`-v, --verbose`
-  Shows verbose output.
+First update the settings in the `crate.yml config file` of the node to:
 
-Examples
---------
-Repurposing a node as a dedicated master node (master: true, data: false)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* ``node.master: true``
+* ``node.data: false``
 
-In this example, a former data node is repurposed as a dedicated master node.
-First update the node’s settings to node.master: true and node.data: false in
-its crate.yml config file. Then run the ``crate-node repurpose`` command to
-find and remove excess shard data:
+.. NOTE::
+
+  Before you continue with the next step, you need to stop CrateDB.
+
+Run ``crate-node repurpose`` to find and remove excess shard data:
 
 .. code-block:: console
 
     node$ ./bin/crate-node repurpose
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
-    Found 2 shards in 2 tables to clean up
-    Use -v to see list of paths and tables affected
-    Node is being re-purposed as master and no-data. Clean-up of shard data will be performed.
+    Found 2 shards in 2 tables to clean up.
+    Use -v to see list of paths and tables affected.
+    Node is being repurposed as master and no-data. Clean-up of shard data will
+    be performed.
 
     Do you want to proceed?
 
-    Confirm [y/N] y
-    Node successfully repurposed to master and no-data.
+    Confirm [y/n] y
```

Comment:
N is uppercase on purpose, since it is the default option when you just press enter. This is common with command line tools. So it should be Confirm [y/N]